net/portmapper/upnp.go

```diff
@@ -44,11 +44,18 @@ type upnpMapping struct {
 	client upnpClient
 }
 
+// NOTE: this must be an upper-case string, or certain routers will reject the
```

Comment:
If already writing such a big doc, start with normal form saying its name and what it is before getting into warnings?